themes/SuiteP/include/ListView/ListViewGeneric.tpl

```diff
@@ -317,3 +324,15 @@ function lvg_nav(m,id,act,offset,t){
 </script>
 <script type="text/javascript" src="include/InlineEditing/inlineEditing.js"></script>
 {/if}
+
+
+{if isset($templateMeta)}
```

Comment:
This allows a developer to add buttons to the list view in the same manner they can for the detail view and the edit view.